app/controllers/spree/retail/shopify/hooks/refunds_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Spree
+  module Retail
+    module Shopify
+      module Hooks
+        class RefundsController < HooksController
+          def create
+            shopify_refund = ShopifyAPI::Refund.find(shopify_refund_id, params: { order_id: shopify_order_id })
+            Spree::Retail::Shopify::GenerateRefundOrder.new(shopify_refund).process
```

Comment:
Right now, there is no easy way to import it separately. 

We could have a task that loops over all the ShopifyOrder and check if there is a refund order associated with it, if there is none then we try to import it but I foresee a lot of problem with that idea. 

I am unsure how much time we want to spend on that.